aws_lambda/policy_checks.py

```diff
@@ -413,9 +413,35 @@ def check_dependabot_enabled(gh: github_interface, repo_url: str) -> bool | str:
     else:
         return f"Error: An error has occured when accessing the API. {dependabot_response}"
 
+def check_point_of_contact(ql: github_graphql_interface, repo_name: str, org: str, codeowners_missing: bool) -> bool:
+    """Checks if a point of contact is defined for a given repository.
+
+    Args:
+        ql (github_graphql_interface): An instance of the github_graphql_interface class to make calls to the GitHub GraphQL API.
+        repo_name (str): The name of the repository to check.
+        org (str): The name of the organisation.
+        codeowners_exists (bool): True if a CODEOWNERS file doesn't exist in the repository.
```

Comment:
Is the comment right? codeowners_exists = True implies codeowners does exist

Reply:
Forgot to update doc string to match passed in parameter name - fixed